source/django-get-started/django-write-data.txt

```diff
@@ -0,0 +1,134 @@
+.. _django-get-started-write:
+
+=====================
+Write Data to MongoDB
+=====================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: app, odm, code example
+
+You can use your application's models to update documents
+stored in the ``sample_mflix`` database. To update documents,
+enter the Python interactive shell and call create, update,
+and delete functions on your model objects.
+
+.. procedure::
+   :style: connected
+
+   .. step:: Start a Python shell
+
+      From your project's root directory, run the following command to 
+      enter the Python shell:
+
+      .. code-block:: bash
+
+         python3 manage.py shell
+
+   .. step:: Import the required classes and modules
+
+      From your Python shell, run the following code to import
+      your models and the module for creating a ``datetime`` object:
+
+      .. code-block:: python
+
+         from sample_mflix.models import Movie, Award, Viewer
+         from django.utils import timezone
+         from datetime import datetime
+
+   .. step:: Insert a movie into the database
+
+      Run the following code to create an ``Movie`` object that
+      stores data about a movie titled ``"Minari"``, including
+      its awards in an ``Award`` object:
+
+      .. code-block:: python
+         
+         movie_awards = Award(wins=122, nominations=245, text="Won 1 Oscar")
+         movie = Movie(title="Minari", plot=
```

Comment:
If you like, it would be more readable to format these using hanging indent, e.g. 
```
movie = Movie(
    title="Minari",
    plot="...",
    ...
)
```
Also, you can use `movie = Movie.objects.create(...)`if you want to avoid the separate `save()` step. That's the usual pattern unless you need to do something between initializing the model and saving it.